start-as-experiment.md

```diff
@@ -0,0 +1,81 @@
+# Title
+
+Start as an Experiment
+
+# Problem
+
+An InnerSource initiative is considered but not started because management is 
+unsure about its outcome and, as a result, is not willing to commit to an 
+investment.
+
+# Context
+
+The majority of software development projects in the company are distributed
```

Comment:
I'm not sure if the global distribution of developers is relevant context for this pattern.  It feels like the following sections would all still apply even without this context.

Reply:
It's a condition of a (the) known case; I think it doesn't do any harm to mention this point, even if the pattern applies without it. (perhaps there could be a parenthetical statement about how the global distribution is not believed to be a necessary condition for applying this pattern?)

Reply:
The way I have rewritten this hopefully makes it clear that global distribution is just one example for a criteria.

Reply:
I think that parenthetical statement would be good.  Wouldn't want anyone to be discouraged from trying out this pattern because they don't believe it applies to them.

Reply:
I agree with Russ that the pattern will apply regardless of whether or not SW development is globally distributed or not. I'll remove it. Thanks for persisting, @rrrutledge !